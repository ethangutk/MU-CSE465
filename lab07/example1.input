z = 3 + (4.3*7**2 - 2)
z
y = -3.141*z^
y