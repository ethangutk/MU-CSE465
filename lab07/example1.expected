ID                 z
LITERAL            =
NUM                3
LITERAL            +
LITERAL            (
NUM                4.3
LITERAL            *
NUM                7
LITERAL            **
NUM                2
LITERAL            -
NUM                2
LITERAL            )
ID                 z
ID                 y
LITERAL            =
NUM                -3.141
LITERAL            *
ID                 z
Invalid character '^' at line 3 column 13
